content/stable-rc.html

```diff
@@ -23,19 +23,14 @@
     <dt>Download multi-platform war file</dt> 
     <dd style="margin-bottom:1em"> 
       <div>
-        <a href="http://mirrors.jenkins-ci.org/war-stable-rc/latest/jenkins.war"><strong>jenkins.war</strong></a>
+        <a href="https://get.jenkins.io/war-stable-rc/latest/jenkins.war"><strong>jenkins.war</strong></a>
```

Comment:
RCs aren’t currently published to get.Jenkins.io, it’s the last remaining work in automated release, (other than more testing)

Reply:
Thanks for catching that.  I'll close this pull request.  We can reopen it after the release candidate builds are being pushed to get.jenkins.io.

Reply:
the rest of the content is good if you do you want to extract it?

Reply:
Fair enough.  I updated the pull request to correct the download link and changed the title to match the content.